pages/platform/kubernetes-k8s/setting-up-a-persistent-volume/guide.en-asia.md

```diff
@@ -0,0 +1,269 @@
+
+For every [Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/){.external} there is a  reclaim policy set by default, that can be changed for individual instances of PV.  On our Cinder based storage classes, **the reclaim policy by default is `Delete`**, as you can verify:
+
+```bash
+kubectl get pv
+```
+
+<pre class="console"><code>$ kubectl 
+get pv
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM              STORAGECLASS        REASON   AGE
+pvc-8ed251c3-4a1e-478e-82cf-01dd637ca459   10Gi       RWO            Delete           Bound    default/test-pvc   cinder-high-speed            15s
+</code></pre>
+
+
+If you delete the PVC, the associated PV is also deleted:
+
+
+```bash
+kubectl delete pvc test-pvc
+kubectl get pvc
+kubectl get pv
+```
+
+
+<pre class="console"><code>$ kubectl 
+$ kubectl delete pvc test-pvc
+persistentvolumeclaim "test-pvc" deleted
+$ kubectl get pvc
+No resources found.
+$ kubectl get pv
+No resources found.
+</code></pre>
+
+
+### Changing the reclaim policy
+
+To illustrate how to change the reclaim policy, let's begin by creating a new PVC using the `test-pvc.yaml` file:
+
+
+```bash
+kubectl apply -f test-pvc.yaml
+```
+
+List the PV and get its name:
+
+
+```bash
+kubectl get pv
+```
+
+And patch it to change its reclaim policy:
+
+```bash
+kubectl patch pv &lt;your-pv-name> -p '{"spec":{"persistentVolumeReclaimPolicy":"Retain"}}'
+```
+
+where &lt;your-pv-name> is the name of your chosen PersistentVolume.
+
+Now you can verify that the PV has the right policy:
+
+```bash
+kubectl get pv
+```
+
+<pre class="console"><code>$ kubectl apply -f test-pvc.yaml
+persistentvolumeclaim/test-pvc created
+$ kubectl get pv
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM              STORAGECLASS        REASON   AGE
+pvc-e53d180d-51a4-4237-b9b3-f6b32cce54cf   10Gi       RWO            Delete           Bound    default/test-pvc   cinder-high-speed            3s
+$ kubectl patch pv pvc-e53d180d-51a4-4237-b9b3-f6b32cce54cf -p '{"spec":{"persistentVolumeReclaimPolicy":"Retain"}}'
+$ kubectl patch pv pvc-e53d180d-51a4-4237-b9b3-f6b32cce54cf -p '{"spec":{"persistentVolumeReclaimPolicy":"Retain"}}'
+persistentvolume/pvc-e53d180d-51a4-4237-b9b3-f6b32cce54cf patched
+$ kubectl get pv
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM              STORAGECLASS        REASON   AGE
+pvc-e53d180d-51a4-4237-b9b3-f6b32cce54cf   10Gi       RWO            Retain           Bound    default/test-pvc   cinder-high-speed            4m
+</code></pre>
+
+In the preceding output, you can see that the volume bound to PVC `default/test-pvc` has reclaim policy `Retain`. It will not be automatically deleted when a user deletes PVC `default/test-pvc`
```

Comment:
You could then try to delete the PVC, and show the PV is still here, but it's not required

Reply:
I am a bit lazy... If I do that, I would need to add a full section on how to deal with those unbounded volumes. We can put that 2nd part in the backlog :)

Reply:
It's ok for me :)